docker/README.md

```diff
@@ -0,0 +1,4 @@
+This directory contains standard images maintained by Istio for test infrastructure purposes.
+
+* prowbazel - gives an environment with git, gcloud, python, bazel, and go. It is suitable for bazel-based projects.
```

Comment:
The reason I ask is because eventually folks would want to run tests against istio/istio on their own environments. Having a jenkins and a prow seems ripe for all sorts of issues such as test environment divergence.